### PR TITLE
[PARTOPS-858] Update QuAC doc with webintent URL

### PR DESCRIPTION
**Pull request by michellechu77**
Updates made per [this thread](https://zapier.slack.com/archives/CH0SN5V0V/p1702396899062609?thread_ts=1702327048.555189&cid=CH0SN5V0V) with Chris, Michelle, and Powell on TPS.

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1702664307407479) and give us a `:review-pr:` emoji.

File: docs/_embed/quick-account-creation.md [hunk 1]
```diff
@@ -43,14 +43,28 @@ If you have an existing Full Zapier Experience or Zap templates element embed, y
 
 ## Add support to the Partner API
 
-1. Retrieve your integration's name from the "title" field returned from the [`/apps` endpoint](https://platform.zapier.com/embed/partner-api#get-v1apps).
-2. Redirect users to the acknowledgement page URL below:
-`https://zapier.com/partner/acknowledgment?sign-up-last-name=<sign-up-last-name>&next=%2Fapp%2Fdashboard&type=quac&name=<Partner Name>&sign-up-first-name=<sign-up=first-name>&sign-up-email=<sign-up-email>`
-3. Replace the following query parameter placeholders in the URL, using the "title" value from step 1 for the "name" parameter:
-- name=`<partner-name>`
+1. Use the URL below to initiate Quick Account Creation, then redirect users into the Editor with the Zap template set in the parameters:
```
Comment (marinahand) [inline docs/_embed/quick-account-creation.md:46]: "Zap editor" instead of "Editor" to follow style guide

**Review by robertscl — APPROVED** [review]

File: docs/_embed/quick-account-creation.md [hunk 2]
```diff
@@ -43,14 +43,28 @@ If you have an existing Full Zapier Experience or Zap templates element embed, y
...
+sign-up-email=<sign-up-email>&
+client-id=<client-id>`
+2. Replace the following query parameter placeholders in the URL:
+- client-id=`<client-id>`
+- template=`<zap-template-id>`
 - sign-up-first-name=`<sign-up-first-name>`
 - sign-up-last-name=`<sign-up-last-name>`
 - sign-up-email=`<sign-up-email>`
+- referer=`<referer>`
+- referrer=`<referrer>`
```
Comment (wavebeem) [inline docs/_embed/quick-account-creation.md:67]: The formatting feels a little hard to follow here

image

and the URL has spaces after each `&` and `?`

Comment (michellechu77) [reply docs/_embed/quick-account-creation.md:67]: oh weird about the spaces - good catch.

Comment (michellechu77) [reply docs/_embed/quick-account-creation.md]: Hope this is better! Matched it with the formatting of URLs & params from the Partner API doc. Thanks for keeping me honest with the formatting!

Comment (MagWeaver) [reply docs/_embed/quick-account-creation.md]: I appreciate the table! Makes it much easier to follow imo

**Review by MagWeaver — APPROVED** [review]
